manifests/charts/base/templates/services.yaml

```diff
@@ -11,6 +11,12 @@ metadata:
   {{- end }}
   namespace: {{ .Release.Namespace }}
 spec:
+  {{- if .Values.pilot.env.ISTIO_DUAL_STACK }}
+  ipFamilyPolicy: RequireDualStack
+  ipFamilies:
+  - IPv6
```

Comment:
Besides the `ipFamilyPolicy` here, we may need to discuss on which IP family should be the first, such as [IPv4, IPv6] or [IPv6, IPv4]. There may be dependencies or decisions on which should be used.